web/core/store/issue/workspace-draft/issue.store.ts

```diff
@@ -0,0 +1,280 @@
+import orderBy from "lodash/orderBy";
+import set from "lodash/set";
+import unset from "lodash/unset";
+import update from "lodash/update";
+import { action, computed, makeObservable, observable, runInAction } from "mobx";
+import { computedFn } from "mobx-utils";
+import {
+  TWorkspaceDraftIssue,
+  TWorkspaceDraftPaginationInfo,
+  TWorkspaceDraftIssueLoader,
+  TWorkspaceDraftQueryParams,
+} from "@plane/types";
+// constants
+import { EDraftIssuePaginationType } from "@/constants/workspace-drafts";
+// helpers
+import { getCurrentDateTimeInISO, convertToISODateString } from "@/helpers/date-time.helper";
+// services
+import workspaceDraftService from "@/services/issue/workspace_draft.service";
+import { IIssueDetail } from "../issue-details/root.store";
+import { clone } from "lodash";
+
+export type TDraftIssuePaginationType = EDraftIssuePaginationType;
+
+export interface IWorkspaceDraftIssues {
+  // observables
+  issuesMap: Record<string, TWorkspaceDraftIssue>;
+  paginationInfo: Omit<TWorkspaceDraftPaginationInfo<TWorkspaceDraftIssue>, "results"> | undefined;
+  loader: TWorkspaceDraftIssueLoader;
+  // computed
+  issueIds: string[];
+  // computed functions
+  getIssueById: (issueId: string) => TWorkspaceDraftIssue | undefined;
+  // helper actions
+  addIssue: (issues: TWorkspaceDraftIssue[]) => void;
+  mutateIssue: (issueId: string, data: Partial<TWorkspaceDraftIssue>) => void;
+  removeIssue: (issueId: string) => void;
+  // actions
+  fetchIssues: (
+    workspaceSlug: string,
+    loadType: TWorkspaceDraftIssueLoader,
+    paginationType?: TDraftIssuePaginationType
+  ) => Promise<TWorkspaceDraftPaginationInfo<TWorkspaceDraftIssue> | undefined>;
+  createIssue: (
+    workspaceSlug: string,
+    payload: Partial<TWorkspaceDraftIssue>
+  ) => Promise<TWorkspaceDraftIssue | undefined>;
+  updateIssue: (
+    workspaceSlug: string,
+    issueId: string,
+    payload: Partial<TWorkspaceDraftIssue>
+  ) => Promise<TWorkspaceDraftIssue | undefined>;
+  deleteIssue: (workspaceSlug: string, issueId: string) => Promise<void>;
+  moveIssue: (workspaceSlug: string, issueId: string, payload: Partial<TWorkspaceDraftIssue>) => Promise<void>;
+  addCycleToIssue: (
+    workspaceSlug: string,
+    issueId: string,
+    cycleId: string
+  ) => Promise<TWorkspaceDraftIssue | undefined>;
+  addModulesToIssue: (
+    workspaceSlug: string,
+    issueId: string,
+    moduleIds: string[]
+  ) => Promise<TWorkspaceDraftIssue | undefined>;
+}
+
+export class WorkspaceDraftIssues implements IWorkspaceDraftIssues {
```

Comment:
This should extend base issues store and use the methods within that for pagination, this looks like it is different from the pattern we have